requirements_test.txt

```diff
@@ -2,20 +2,21 @@ captum==0.4.1
 coverage==6.2
 flake8==4.0.1
 matplotlib==3.3.4
-numpy==1.19.5
+numpy==1.23.2
 opencv-python==4.5.5.62
 protobuf~=3.19.0
 pytest==6.2.5
 pytest-cov==3.0.0
 pytest-lazy-fixture==0.6.3
 scikit-image==0.19.1 ; python_version > '3.6'
-scikit-image==0.17.2 ; python_version == '3.6'
 scikit-learn==0.24.2
 scipy==1.5.4
-tensorflow==2.6.2
+tensorflow==2.9.1
```

Comment:
Why is only `requirements_test.txt` and not `requirements.txt` updated?

Reply:
Don't we e.g., need jax for both (also not sure about whether we want to include jax but let's discuss for performance gains )

Reply:
I did not see `requirements.txt` being used by anything. I will also update it

Reply:
updated

Reply:
ok thanks